component/Applaud/component/ApplaudTimeline.jsx

```diff
@@ -17,7 +17,7 @@ function ApplaudTimeline({ list }) {
             </div>
           }
           className="recent-activity-timeline"
-          key={item.id + index + "activity"}
+          key={item.id + "activity" + index}
```

Comment:
Can we use ` key={item.id}`  instead of this `key={item.id + "activity" + index}` ?